src/screenbot.js

```diff
@@ -49,22 +91,31 @@ var Screenbot = (function Screenbot() {
         _this.command = function (command, cb) {
           console.log("Source: " + _private.source);
 
-          _request(_endpoint(command), function(success, response) {
-
-            if (!success) return cb(false, response.error);
+          _request(_endpoint(command), function(err, result) {
+            if(err instanceof Error) return cb(err);
+            if(err) return cb(_handleError({ code: ERROR_CODES.CLIENT_UNAVAILABLE.code, message: err.error }));
```

Comment:
Following the other comments, this could be written with just passing the error as the second argument:
```
if(err) return cb(new ScreenbotError(CLIENT_UNAVAILABLE, err));
```